fudy/NoteScreen/Component/TaskItem.qml

```diff
@@ -18,7 +18,7 @@ Rectangle {
 	signal removeCurrentItem()
 	signal enabledEditted(bool enabled)
 	signal headerEditted(string headerText)
-	signal noteTextEditted()
+	signal noteTextEditted(string noteText)
```

Comment:
QString

Reply:
I think you have a confusion between C++ and QML syntax. Please check again.